app/components/searchworks4/record_summary_component.rb

```diff
@@ -10,7 +10,7 @@ def initialize(presenter:)
     end
 
     def authors
-      return presenter.document.eds_authors if presenter.document.eds?
+      return presenter.document.eds_authors.uniq if presenter.document.eds?
```

Comment:
```suggestion
      return presenter.document.eds_authors&.uniq if presenter.document.eds?
```

It looks like eds_authors can be nil?
https://github.com/sul-dlss/SearchWorks/blob/e360df830f40ef289e96870442264c95b0670971/app/models/eds_document.rb#L138